cache_test.go

```diff
@@ -23,36 +23,36 @@ func test(t *testing.T, storeFn func() CacheStore) {
 			t.Fatal("flights before Update should return empty RedisMessage and non-nil CacheEntry")
 		}
 
-		store.Delete([]RedisMessage{{typ: '+', string: "key"}}) // Delete should not affect pending CacheEntry
+		store.Delete([]RedisMessage{redisMessageContainString('+', "key")}) // Delete should not affect pending CacheEntry
```

Comment:
```suggestion
		store.Delete([]RedisMessage{strmsg('+', "key")}) // Delete should not affect pending CacheEntry
```
I prefer a shorter function name.